docs/source/utils/notebook.py

```diff
@@ -77,17 +77,17 @@ class DocumentationLink(ReplacePattern):
     USAGE EXAMPLES
     --------------
 
-    [link](%doclink(api,script.core.script))
+    [Node](%doclink(api,script.core.script)) -> [Node](http://github.com/deeppavlov/dialog_flow_framework/tree/dev/dff/script/core/script#Node)
 
-    [link](%doclink(api,script.core.script,Node))
+    [Node](%doclink(api,script.core.script,Node)) -> [Node](http://github.com/deeppavlov/dialog_flow_framework/tree/dev/tutorials/messengers/web_api_interface/4_streamlit_chat)
 
-    [link](%doclink(tutorial,messengers.web_api_interface.4_streamlit_chat))
+    [API-configuration](%doclink(tutorial,messengers.web_api_interface.4_streamlit_chat)) -> [API-configuration](http://github.com/deeppavlov/dialog_flow_framework/tree/dev/tutorials/messengers/web_api_interface/4_streamlit_chat#API-configuration)
 
-    [link](%doclink(tutorial,messengers.web_api_interface.4_streamlit_chat,API-configuration))
+    [API-configuration](%doclink(tutorial,messengers.web_api_interface.4_streamlit_chat,API-configuration)) -> [API-configuration](http://github.com/deeppavlov/dialog_flow_framework/tree/dev/tutorials/messengers/web_api_interface/4_streamlit_chat#API-configuration)
 
-    [link](%doclink(guide,basic_conceptions))
+    [basic_conceptions](%doclink(guide,basic_conceptions)) -> [basic_conceptions](http://github.com/deeppavlov/dialog_flow_framework/tree/dev/docs/sources/user_guides/basic_conceptions)
 
-    [link](%doclink(guide,basic_conceptions,example-conversational-chat-bot))
+    [example-conversational-chat-bot](%doclink(guide,basic_conceptions,example-conversational-chat-bot)) -> [example-conversational-chat-bot](http://github.com/deeppavlov/dialog_flow_framework/tree/dev/docs/sources/user_guides/basic_conceptions#example-conversational-chat-bot)
```

Comment:
These are just examples!

Reply:
`%doclink(api,script.core.script)` points to the `Script`  module. I think it would make more sense if it was
`[Script module](%doclink(api,script.core.script))` or `[dff.script.core.script](%doclink(api,script.core.script))`
instead of `[Node](%doclink(api,script.core.script))`.

The same with `%doclink(tutorial,messengers.web_api_interface.4_streamlit_chat)`. `API-configuration` is just one of the subsections of that tutorial, whereas the link points to the tutorial itself.